build.gradle

```diff
@@ -202,14 +202,14 @@ def versions = [
   springCloud     : '2020.0.1',
   springfoxSwagger: '3.0.0',
   testcontainers  : '1.15.2',
-  jetty           : '9.4.43.v20210629',
+  jetty           : '9.4.49.v20220914',
   netty           : '4.1.77.Final'
 ]
 
 ext['spring-framework.version'] = '5.3.21'
 ext['spring-security.version'] = '5.7.5'
 ext['log4j2.version'] = '2.17.1'
-ext['jackson.version'] = '2.13.2'
+ext['jackson.version'] = '2.13.4'
```

Comment:
@JamesCollettCGI upgrade Jackson to 2.14.0-rc1 to fix CVE-2022-22970 and few other CVEs as well.

Reply:
Updated Jackson to 2.14.0-rc1